core/domain/collection_domain_test.py

```diff
@@ -198,9 +198,6 @@ def test_skill_ids_validation(self):
                 'question_ids': []
             })}
 
-        self._assert_validation_error(
```

Comment:
Hang on -- the correct way to handle this is generally _not_ to delete the test. I think the error message just needs to be changed.

Could you reinstate the check please?